services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -186,6 +188,20 @@ async def stop_dynamic_service(
 
         return RedirectResponse(str(redirection_url))
 
+    # Serice was marked for removal, the scheduler will
+    # take care of stopping cleaning up all allocated resources:
+    # services, containsers, volumes and networks.
+    # Once the service is no longer being tracked this can return
+    dynamic_sidecar_settings: DynamicSidecarSettings = (
+        request.app.state.settings.DYNAMIC_SERVICES.DYNAMIC_SIDECAR
+    )
+    _STOPPED_CHECK_INTERVAL = 1.0
+    async with async_timeout.timeout(
+        dynamic_sidecar_settings.DYNAMIC_SIDECAR_WAIT_FOR_SERVICE_TO_STOP
```

Comment:
Make sure that the client does not have a timeout smaller than this number.

Reply:
Calling service has an adequate timeout 👍